pkg/minikube/constants/constants.go

```diff
@@ -29,8 +29,9 @@ const MachineName = "minikubeVM"
 // APIServerPort is the port that the API server should listen on.
 const APIServerPort = 8443
 
-// Fix for windows
-var Minipath = filepath.Join(homedir.HomeDir(), ".minikube")
+// Fix for windows: the home dir contains forward slashes, which results
```

Comment:
Shouldn't this be "contains back slashes"? This function is covering all os file path separators to forward slashes to make url.Parse happy, right?

This probably makes sense to convert only where needed, rather than here where an os formatted file path is fine.